api/src/main/java/org/pmiops/workbench/cdr/dao/CustomCBCriteriaDao.java

```diff
@@ -4,45 +4,44 @@
 import org.pmiops.workbench.cdr.model.DbCardCount;
 import org.pmiops.workbench.cdr.model.DbCriteria;
 import org.springframework.data.domain.Page;
-import org.springframework.data.domain.PageRequest;
 import org.springframework.data.domain.Pageable;
 
 public interface CustomCBCriteriaDao {
-  Page<DbCriteria> findCriteriaByDomainAndStandardAndNameEndsWith(
-      String domain, Boolean standard, List<String> endsWithList, Pageable page);
+  Page<DbCriteria> findCriteriaByDomainAndNameEndsWithAndStandard(
+      String domain, List<String> endsWithList, Boolean standard, Pageable page);
 
-  Page<DbCriteria> findCriteriaByDomainAndStandardAndTermAndNameEndsWith(
-      String domain, Boolean standard, String term, List<String> endsWithList, Pageable page);
+  Page<DbCriteria> findCriteriaByDomainAndNameEndsWithAndTermAndStandard(
+      String domain, String term, List<String> endsWithList, Boolean standard, Pageable page);
 
   List<DbCriteria> findCriteriaByDomainAndTypeAndStandardAndNameEndsWith(
-      String domain, String type, Boolean standard, List<String> endsWithList, PageRequest page);
+      String domain, String type, Boolean standard, List<String> endsWithList, Pageable page);
```

Comment:
Using Pageable interface instead of PageRequest class.

Reply:
hmm...yep interface is better